### PR TITLE
Consolidate intro to camunda

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Closes https://github.com/camunda/camunda-docs/issues/4011 -- consolidates introduction to Camunda pages between guides and components.

## Notes

- **Will backport upon approval.**
- Do we feel Connectors are highlighted well enough? I don't think we consider Connectors one of the six "components," and it is mentioned in next steps, but not sure if I should add an admonition noting Connectors in the intro to the page.
- Does this location make sense? It is centered around the guides content, so it makes sense with the current changes, but I'm open to adjusting this based on opinions.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by akeller** [comment]
> Do we feel Connectors are highlighted well enough? I don't think we consider Connectors one of the six "components," and it is mentioned in next steps, but not sure if I should add an admonition noting Connectors in the intro to the page.
Does this location make sense? It is centered around the guides content, so it makes sense with the current changes, but I'm open to adjusting this based on opinions.

Thank you for the notes ❤️ really helps focus what you want us to review!

**Comment by akeller** [comment]
Updating the branch to see if I can resolve the build errors and get the preview environment running for review.

**Comment by akeller** [comment]
@christinaausley can you have a look at the build errors?

File: docs/guides/introduction-to-camunda-8.md [hunk 1]
```diff
@@ -6,44 +6,92 @@ slug: /guides/
...
 
-1. [Create a Camunda 8 account](./create-account.md) - Create a Camunda 8 account to create clusters, deploy processes, and create a new instance.
-2. [Model your first process](./model-your-first-process.md) - Design and deploy a process, and share the process with your teammates or other stakeholders to begin collaborating.
-3. [Create a cluster](./create-cluster.md) - Create your first cluster to execute processes.
-4. [Orchestrate human tasks](./getting-started-orchestrate-human-tasks.md) - Assign human tasks to users so they can enter the necessary data to drive the business process forward.
-5. [Get started with API orchestration](./getting-started-orchestrate-apis.md) - Configure Connector tasks with domain-specific parameters.
-6. [Orchestrate microservices](./getting-started-orchestrate-microservices.md) - Orchestrate the microservices necessary to achieve your end-to-end automated business process.
+<DocCardList items={[{type:"link", href:"/docs/next/guides/getting-started/", label: "Step 1: Create a Camunda 8 account", docId:"guides/getting-started"},
```
Comment (akeller) [inline docs/guides/introduction-to-camunda-8.md:53]:
```suggestion
<DocCardList items={[{type:"link", href:"/docs/next/guides/create-account/", label: "Step 1: Create a Camunda 8 account", docId:"guides/create-account"},
```

`guides/getting-started/` is the slug for this page. I think you want the page id in both places.

File: docs/guides/introduction-to-camunda-8.md [hunk 2]
```diff
@@ -6,44 +6,92 @@ slug: /guides/
...
-- [Tasklist](/components/tasklist/introduction-to-tasklist.md) - Complete tasks which require human input.
-- [Operate](/components/operate/operate-introduction.md) - Manage, monitor, and troubleshoot your processes.
-- [Optimize]($optimize$/components/what-is-optimize) - Improve your processes by identifying constraints in your system.
+Additionally, Camunda hosts a section of [Best Practices](/components/best-practices/best-practices-overview.md). A mix of conceptual and practical implementation information, this section hosts our condensed experience using BPMN and DMN on the Camunda tool stack collected by consulting engagement with our customers and feedback from the community.
 
-For more conceptual information about Camunda 8, see [What is Camunda 8](components/concepts/what-is-camunda-8.md).
-For an on-demand demonstration of the product, visit [Camunda Academy](https://bit.ly/3CvooTX).
+:::note Looking for deployment guides?
+
+Deployment guides for Camunda 8 components are available in the [Self-Managed section]/self-managed/about-self-managed.md.
```
Comment (akeller) [inline docs/guides/introduction-to-camunda-8.md:41]:
```suggestion
Deployment guides for Camunda 8 components are available in the [Self-Managed section](/self-managed/about-self-managed.md).
```

**Review by akeller — COMMENTED** [review]
In the last section (**Learn more**), I'm a bit overwhelmed with doc cards. Maybe we can highlight a few as doc cards and show the rest as links or something less prominent.

Similarly, I think the **Getting started** section highlights our long time tension between how to get started with the platform, holistically, and how to do specific use cases. One recommendation here would be to call attention to the two "main" guides - Java Spring & modeling (maybe as doc cards?), while also mentioning the use case guides (maybe as links).

**Comment by christinaausley** [comment]
@camunda/tech-writers @akeller I think this PR is a fine baseline and starting point, but I should have started conversation on a Miro board before launching changes -- I'm going to convert this to a draft, add a `hold` label, and add a link to that Miro board here shortly for continued discussion.

**Comment by christinaausley** [comment]
@akeller @conceptualshark [here is the Miro board](https://miro.com/app/board/uXjVKm1QkNs=/?share_link_id=898204352738) -- as @mesellings is out next week I figured the two of you may be best for review 👍 

Once finalized I can pull these changes into this existing PR.

**Comment by akeller** [comment]
Planning to review the Miro board today.

**Comment by akeller** [comment]
I reviewed the Miro and then returned to the PR - I like where you are going with these changes! I particularly like that you put this in the Guides section, where the page structure mimics the sidebar content. 

One comment I left on Miro that I want to elaborate on a bit more - flipping the order of the doc cards and presenting the modeling content before the Spring SDK content. This is mostly a comment based on an untested hypothesis that our non-dev personas might be intimidated by seeing SDK-first content... well, first! We know nearly every persona interacting with Camunda must model (or interpret/see a model). We would also want to flip the order of the content on the sidebar. Do you have thoughts on this?

**Comment by christinaausley** [comment]
@akeller @conceptualshark Thank you for your quick feedback! I'll plan to polish up this PR today with the proposed changes.

> One comment I left on Miro that I want to elaborate on a bit more - flipping the order of the doc cards and presenting the modeling content before the Spring SDK content.

I agree with this -- I think we should flip the content as the Modeling content is so entry-level and core to the Camunda experience as a beginner. At least, it's the first thing I would click on as a newbie trying to figure out what Camunda is all about. I'm happy to swap it in the sidebar in this PR, but agreed that this should switch too.

**Review by akeller — APPROVED** [review]
I like the consolidation! 

🙈 but of course, now I'm seeing other things I want to fix outside the scope of this PR. I'll save that for another issue scope.

**Comment by christinaausley** [comment]
> I like the consolidation!
> 
> 🙈 but of course, now I'm seeing other things I want to fix outside the scope of this PR. I'll save that for another issue scope.

@akeller Feel free to let it fly in https://github.com/camunda/camunda-docs/issues/3199 😬 

Merging this!

**Pull request merged**